app/services/ingest_ephemera_csv.rb

```diff
@@ -80,7 +80,7 @@ def attributes
   # rubocop:enable Metrics/MethodLength
 
   def files
-    @files || Dir.glob("#{image_path}/*.{TIF,TIFF,tif,tiff,jpg,jpeg,png}").sort.map do |file|
+    @files || Dir.glob("#{image_path}/*.{tif,tiff,jpg,jpeg,png}").sort.map do |file|
```

Comment:
Looks like this deletion is causing the test failure. Maybe downcasing `image_path` to avoid having to have case-sensitive matching would be a better approach?

Reply:
Agreed; used File::FNM_CASEFOLD.